pages/clan.php

```diff
@@ -116,6 +120,61 @@ function clan() {
 			$system->printMessage();
 		}
 	}
+    //Kick Controls
+    if($page == 'kick' && $_GET['kick']) {
```

Comment:
TBH I think we can just do auto kick and leave out the manual kick option. If we think it's okay for users to remove someone after a period of time we should just proactively remove them